services/mgmt/vmware/src/tests.rs

```diff
@@ -0,0 +1,47 @@
+use super::*;
+
+#[cfg(feature = "package-2023-03-01")]
```

Comment:
Instead of src/tests.rs, why not have tests/foo.rs?  Based on my understanding of the rest of the code generation, this wouldn't need anything special to manage.

Reply:
That is a lot easlier.